app/src/main/res/values/strings.xml

```diff
@@ -87,6 +87,8 @@ Unfortunately there is no faucet for the test-net. So if you want to try out thi
     <string name="input_hint_notes">notes</string>
     <string name="input_hint_account_name">account name</string>
     <string name="please_enter_name">Please enter a name</string>
+    <string name="info">About</string>
```

Comment:
Lets call it info not About - is more an about currently - but just as there is some text still to be written ;-)
Calling it About now will confuse translators and lead to bad translations in the end - soon need to do the transifex thing